content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -115,7 +115,10 @@ detail the structure of that `.status` field, and its content for each different
 
 ## {{% heading "whatsnext" %}}
 
-* Learn about the most important basic Kubernetes objects, such as [Pod](/docs/concepts/workloads/pods/).
-* Learn about [controllers](/docs/concepts/architecture/controller/) in Kubernetes.
-* [Using the Kubernetes API](/docs/reference/using-api/) explains some more API concepts.
+Learn more about the following.
```

Comment:
nit
```suggestion
Learn more about the following:
```

Reply:
Hi @reylejano,
Thanks for your valuable suggestion.

i have incorporated the above suggestion.